adam-core/src/main/scala/org/bdgenomics/adam/rdd/read/MDTagging.scala

```diff
@@ -71,8 +71,20 @@ private[read] case class MDTagging(
         contig <- Option(read.getContigName)
         if read.getReadMapped
       } yield {
-        maybeMDTagRead(read, referenceFileB.value
-          .extract(ReferenceRegion.unstranded(read)))
+        try {
+          maybeMDTagRead(read, referenceFileB.value
+            .extract(ReferenceRegion.unstranded(read)))
+        } catch {
+          case t: Throwable => {
+            if (validationStringency == ValidationStringency.STRICT) {
```

Comment:
I'm +0. I don't love ValidationStringency, but I'm a bit too lazy to deconvolve it from our codebase. I guess I'd say that it is on the sorted list of refactors that I'd like to do, but lower down on the list. That being said, it's probably not that much work. CC @ryan-williams for thoughts as well.